Concept/content/Screen01.ui.qml

```diff
@@ -1,71 +1,148 @@
-/*
-This is a UI file (.ui.qml) that is intended to be edited in Qt Design Studio only.
-It is supposed to be strictly declarative and only uses a subset of QML. If you edit
-this file manually, you might introduce QML code that is not supported by Qt Design Studio.
-Check out https://doc.qt.io/qtcreator/creator-quick-ui-forms.html for details on .ui.qml files.
-*/
-
 import QtQuick 6.2
 import QtQuick.Controls 6.2
 import Concept
 
 Rectangle {
-    id: rectangle
     width: Constants.width
     height: Constants.height
 
-    color: Constants.backgroundColor
+    // This is the title bar
+    Rectangle {
+        id: nameBar
+        width: parent.width
+        height: 80
+        color: "#A1B5D8"
+
+        //Title Concept
+        Text {
+            text: "Concept"
+            id: conceptText
+            font.pixelSize: 50
+            //font.family: "Helvetica"
+            anchors.left: parent.left
+            verticalAlignment: Text.AlignVCenter
+            leftPadding: 150
+        }
+
+        //spacing item
+        Item {
+            id: spacerTitleHelp
+            anchors.left: conceptText.right
+            width: 800
+        }
+
+        // "Help" button
+        Button {
+            id: helpButton
+            text: "Help"
+            font.pixelSize: 20
+            anchors.left: spacerTitleHelp.right
+            anchors.verticalCenter: parent.verticalCenter
+        }
 
-    Button {
-        id: button
-        text: qsTr("Press me")
-        anchors.verticalCenter: parent.verticalCenter
-        checkable: true
-        anchors.horizontalCenter: parent.horizontalCenter
+        //spacing item
+        Item {
+            id: spacerHelpSearch
+            anchors.left: helpButton.right
+            width: 100
+        }
 
-        Connections {

```

Comment:
this connection to the button won't be useful, for now it only starts the animation to change the background when the button is pressed